tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -2389,4 +2389,29 @@ public function downloadedJsonContentShouldContainKeyUserAndMatch(PyStringNode $
 			$this->featureContext->getJSONSchema($schemaString)
 		);
 	}
+
+	/**
+	 * @When user :user tries to export GDPR report of user :ofUser to :path using Graph API
+	 *
+	 * @param string $user
+	 * @param string $ofUser
+	 * @param string $path
+	 *
+	 * @return void
+	 *
+	 */
+	public function userTriesToExportGdprReportOfAnotherUserUsingGraphApi(string $user, string $ofUser, string $path): void {
+		$credentials = $this->getAdminOrUserCredentials($user);
+		$userId = $this->featureContext->getAttributeOfCreatedUser($ofUser, 'id');
+		$this->featureContext->setResponse(
+			GraphHelper::generateGDPRReport(
+				$this->featureContext->getBaseUrl(),
+				$this->featureContext->getStepLineRef(),
+				$credentials['username'],
+				$credentials['password'],
+				$userId,
```

Comment:
```suggestion
				$this->featureContext->getAttributeOfCreatedUser($ofUser, 'id'),
```
may be we can use it like this